packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -1,116 +1,90 @@
-import { CalendarDate } from '@internationalized/date';
-import { useRef } from 'react';
+import { DatePicker, type DateValue } from 'react-aria-components';
+import type RAC from 'react-aria-components';
 
-import { AriaDatePickerProps, useDatePicker } from '@react-aria/datepicker';
-import { mergeProps } from '@react-aria/utils';
-
-import { useDatePickerState } from '@react-stately/datepicker';
-
-import { WidthProp, useClassNames, useStateProps } from '@marigold/system';
+import { WidthProp, useClassNames } from '@marigold/system';
 
 import { Button } from '../Button';
-import { Calendar } from '../Calendar/Calendar';
-import { DateField } from '../DateField';
-import { Popover } from '../Overlay';
+import { Calendar } from '../Calendar';
+import { DateInput } from '../DateField/DateInput';
+import { FieldBase, FieldBaseProps } from '../FieldBase/_FieldBase';
+import { Popover } from '../Overlay/Popover';
+
+type RemovedProps =
+  | 'isDisabled'
+  | 'isReadOnly'
+  | 'isRequired'
+  | 'isInvalid'
+  | 'style'
+  | 'className'
+  | 'isOpen';
 
-// Props
-// -----------------
 export interface DatePickerProps
-  extends Omit<
-    AriaDatePickerProps<CalendarDate>,
-    'isDisabled' | 'isRequired' | 'isReadOnly' | 'isOpen'
-  > {
-  disabled?: boolean;
-  required?: boolean;
-  readonly?: boolean;
-  open?: boolean;
-  error?: boolean;
-  shouldCloseOnSelect?: boolean;
+  extends Omit<RAC.DatePickerProps<DateValue>, RemovedProps>,
+    Pick<FieldBaseProps<'label'>, 'label' | 'description' | 'errorMessage'> {
+  disabled?: RAC.DatePickerProps<DateValue>['isDisabled'];
+  required?: RAC.DatePickerProps<DateValue>['isRequired'];
+  readOnly?: RAC.DatePickerProps<DateValue>['isReadOnly'];
+  error?: RAC.DatePickerProps<DateValue>['isInvalid'];
+  open?: RAC.DatePickerProps<DateValue>['isOpen'];
   variant?: string;
   size?: string;
   width?: WidthProp['width'];
 }
 
-// Component
-// -----------------
-export const DatePicker = ({
+const _DatePicker = ({
   disabled,
   required,
-  readonly,
-  open,
+  readOnly,
   error,
-  shouldCloseOnSelect,
   variant,
   size,
-  width,
+  value,
```

Comment:
According to my other comment from packages/components/src/DatePicker/DatePicker.stories.tsx. The value prop is destructured here and nowhere used anymore. So if we remove that the default value should work with only the **value** prop in a controlled state. Alternatively we can pass it to Fieldbase in l.61 but I think it isn't needed.

Reply:
oh thank you :D resolved